tests/test_gcloud_client.py

```diff
@@ -46,11 +47,17 @@ def set_iam_policy(self, policy):
 def test_upload_xlsx_to_gcs(monkeypatch):
     monkeypatch.setattr("google.cloud.storage.Client", MagicMock)
 
-    trial = "10021"
+    trial = "test_trial"
     template_type = "assay"
     assay_type = "wes"
 
-    uri = upload_xlsx_to_gcs(trial, template_type, assay_type, BytesIO(b"12345"))
+    uri = upload_xlsx_to_gcs(
+        trial,
+        template_type,
+        assay_type,
+        BytesIO(b"12345"),
+        datetime.datetime.now().isoformat(),
+    )
     assert trial in uri
```

Comment:
This and the below assertions will need to be updated now that `upload_xlsx_to_gcs` returns a `Blob`, not a URI.